tests/unit/statistical_testing/QuadraticTimeMMD_unittest.cc

```diff
@@ -354,7 +354,7 @@ TEST(QuadraticTimeMMD, perform_test_permutation_biased_full)
 	// assert against local machine computed result
 	mmd->set_statistic_type(ST_BIASED_FULL);
 	float64_t p_value=mmd->compute_p_value(mmd->compute_statistic());
-	EXPECT_NEAR(p_value, 0.0, 1E-10);
+	EXPECT_NEAR(p_value, 0.8, 1E-10);
```

Comment:
probably not because of change of seed then... hence my question here as i didn't want to dig into what is actually happening in this unit test.
i.e. this part needs to be reverted and figure out where else things get not properly done with the random.